src/components/DataDictionary.tsx

```diff
@@ -90,7 +90,11 @@ export const columns: ColumnDescriptor[] = [
     name: "category",
     label: "Category",
     rowValueDescriptor: "category",
-    formatter: (x: any, isRow: boolean, field: ILookmlModelExploreField) => {
+    formatter: function CategoryLabelFormatter(
```

Comment:
did the linter request a named keyword function here? The convention is to use arrow functions for React components, as in the other `formatter` fields in this array.